Source/santad/EventProviders/SNTEndpointSecurityFileAccessAuthorizer.mm

```diff
@@ -481,6 +485,27 @@ - (FileAccessPolicyDecision)handleMessage:(const Message &)msg
                               targetPathCopy, policyDecision);
                           }];
     }
+
+    if (!optionalPolicy.value()->silent && self.fileAccessBlockCallback) {
```

Comment:
nit: This will always be false.

Reply:
As discussed, on purpose for now as noted in `WatchItemPolicy.h`